### PR TITLE
Admin into system

### PR DESCRIPTION
**Pull request by filipefurtad0**
#### What? Why?

Continues #7299

Updates/moves several admin/enterprise feature specs into system.

#### What should we test?

Green build.

#### Release notes

Updates/moves several admin/enterprise feature specs into system.

Changelog Category: Technical changes

### CODE REVIEW

File: spec/base_spec_helper.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
 require 'simplecov' if ENV["COVERAGE"]
 require 'rubygems'
 require 'pry' unless ENV['CI']
+require 'view_component/test_helpers'
```
Comment (Matt-Yorkley) [inline spec/base_spec_helper.rb:10]: The tests don't work without this? :thinking:

File: spec/system/admin/product_import_spec.rb [hunk 2]
```diff
@@ -715,7 +715,7 @@ def save_data
 
   def expect_progress_percentages(*percentages)
     percentages.each do |percentage|
-      expect(page).to have_selector ".progress-interface", text: percentage
+      page.has_selector? ".progress-interface", text: percentage # Waits for progress bar
```
Comment (Matt-Yorkley) [inline spec/system/admin/product_import_spec.rb:718]: I think in this case `expect` will also wait, so this change isn't needed.

File: spec/base_spec_helper.rb (hunk 1 above)
Comment (filipefurtad0) [reply spec/base_spec_helper.rb:10]: I could not figure out why `order_print_ticket_spec.rb` was failing after switching to the `system_spec` helper. By comparing feature and system spec I've found this one bit to be missing, and it seems to bring it to green, but I can't explain it either... Without this line it fails with this error:

`rspec ./spec/features/admin/order_print_ticket_spec.rb:36 # `


```
    As an administrator
    I want to print a ticket for an order
 as an enterprise manager viewing the edit page can print an order's ticket
     Failure/Error: print_data = page.evaluate_script('printData');
     
     Ferrum::JavaScriptError:
       ReferenceError: printData is not defined
           at <anonymous>:1:15

```

File: spec/system/admin/product_import_spec.rb (hunk 2 above)
Comment (filipefurtad0) [reply spec/system/admin/product_import_spec.rb:718]: I could not get it to pass locally without this change, I get:

```
  1) Product Import handling a large file (120 data rows) when importing to product list validates and saves all batches
     Failure/Error: expect(page).to have_selector ".progress-interface", text: percentage
       expected to find visible css ".progress-interface" with text "33%" but there were no matches. Also found "", "Save imported products (0%)", which matched the selector but not all filters. 
```

**Review by Matt-Yorkley — APPROVED** [review]

**Pull request merged**